### DISCUSSION TITLE
Adding or removing the default instructions for use message

### DISCUSSION THREAD

**Discussion by JoelTapia99** [SBoudrias/Inquirer.js#1121] [Q&A] [answered]
is there a way to edit or delete the red box message?
I tried using property **prefix** and **suffix**, but it's not what I need.

Thank you

![image](https://user-images.githubusercontent.com/58013001/171550147-3911de79-49c3-49b1-a019-4ac237bce7bc.png)

**Comment by SBoudrias** [comment, marked as answer]
This isn't possible at the moment.

I'd like to add better theming/localization capabilities, but that is honestly far ahead.

**Reply by JoelTapia99** [reply]
I understand, thank you very much

**Comment by IdkMan2Usertive** [comment]
There should be an option to hide those instructions at least, as for the checkbox field they are too long:

![image](https://user-images.githubusercontent.com/82803183/218499688-de15aea6-a5f3-4a36-894f-29a5cbb5ce91.png)

I am building a CLI tool for my own team where everyone knows how to play with interactive CLI, they don't need any instructions.
